### PR TITLE
Feat/pericope translation

### PR DESCRIPTION
**Pull request by ForeverYoung1208**

### CODE REVIEW

File: frontend/src/components/pericope-translations/PericopeTrPage.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    nation_id: string;
+    language_id: string;
+  }> {}
+
+export const PericopeTrPage: React.FC<PericopeTrPageProps> = ({
+  match,
+}: PericopeTrPageProps) => {
+  const { nation_id, language_id } = match.params;
+  const { tr } = useTr();
+  const router = useIonRouter();
```
Comment (lucky-gru) [inline frontend/src/components/pericope-translations/PericopeTrPage.tsx]: Hi @ForeverYoung1208 ,
useIonRouter and useHistory is similar hooks.
Hope to use useHistory

File: frontend/src/components/common/DocumentList/DocumentItem.tsx
Comment (lucky-gru) [inline frontend/src/components/common/DocumentList/DocumentItem.tsx:1]: Hi @ForeverYoung1208 
Please don't move `DocumentList` to the common folder.
`DocumentList` is not a common. It is use their graphql hooks and another things.
Must exist in the `documents/DocumentList` folder.

File: api/src/components/pericopies/pericopies.service.ts [hunk 2]
```diff
@@ -239,4 +239,14 @@ export class PericopiesService {
       },
     };
   }
+
```
Comment (lucky-gru) [inline api/src/components/pericopies/pericopies.service.ts:246]: @ForeverYoung1208 
FYI, getRecommendedPericopiesByDocumentId and getPericopiesTexts should have a after parameter for the pagination.

the after parameter is a type of string data

```
JSON.stringify({document_id: number, page: number})
```

The reason after parameter having a `document_id` is for graphql cache.

Comment (lucky-gru) [reply api/src/components/pericopies/pericopies.service.ts:246]: The `document_word_entry` data is stored out of order, so you should calculate the order using dfs or another method.

`components/documents/document-word-entries.service.ts/sortBlockEntries()`

**Comment by ForeverYoung1208** [comment]
Yet it is not finished (still need to finish frontend and maybe fix some bugs), but i think it is better to merge to not collect merge conflicts.

**Comment by lucky-gru** [comment]
Hi Ihor, plz resolve above reviews

**Comment by ForeverYoung1208** [comment]
> Hi Ihor, plz resolve above reviews

@lucky-gru  Hmm, i can't find any. Can you pls give me a link or just tip me where i can find your comments?

**Review by lucky-gru — APPROVED** [review]

**Pull request merged**